templates/collection_view_screen/app/screens/name_screen.rb

```diff
@@ -0,0 +1,59 @@
+class <%= @name_camel_case %>Screen < UICollectionViewController
+  include ProMotion::ScreenModule
+
+  title "Your title here"
+  stylesheet <%= @name_camel_case %>ScreenStylesheet
+
+  <%= @name.upcase %>_CELL_ID = "<%= @name_camel_case %>Cell"
+
+  def self.new(args = {})
+    # Set layout
+    layout = UICollectionViewFlowLayout.alloc.init
+    s = self.alloc.initWithCollectionViewLayout(layout)
+    s.screen_init(args) if s.respond_to?(:screen_init)
+    s
+  end
+
+  def on_load
+    collectionView.tap do |cv|
+      cv.registerClass(ImagesCell, forCellWithReuseIdentifier: IMAGES_CELL_ID)
```

Comment:
We'll just want to put the same identifier here as you have defined above
`<%= @name.upcase %>_CELL_ID`